app/src/playstore/java/openfoodfacts/github/scrachx/openfood/utils/CameraUtils.kt

```diff
@@ -0,0 +1,122 @@
+@file:Suppress("DEPRECATION")
```

Comment:
```suggestion
```

Let's keep that for the methods that need it

Reply:
@VaiTon  It will show a deprecation warning on import.